src/plan_bot.py

```diff
@@ -269,7 +282,7 @@ def process_post(
     if match:
         print("plan match: ", plan_id, post.id, plan_confidence)
 
-        reply_string = build_response_text(plan, post)
+        reply_string = build_response_text(plan, post, verbatim=verbatim)
```

Comment:
Instead of extending this function with a flag, if you make `why_warren` an "operation", you can just use the operations_map above!